PPAK_CVS/Sources/Extensions/UIColor.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  UIColor.swift
+//  PPAK_CVS
+//
+//  Created by hyeonseok on 2022/09/26.
+//
+
+import Foundation
+import UIKit
```

Comment:
UIKit 안에 Foundation이 있습니다!!!!!

```suggestion
import UIKit
```